dns/zone.py

```diff
@@ -500,16 +501,27 @@ def to_file(self, f, sorted=True, relativize=True, nl=None):
 
         str_type = string_types
```

Comment:
OK, I didn't want to touch it because it is unrelated, but I will if you insist :)